include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -474,9 +478,9 @@ struct OrtApi {
   /// @{
 
   /** \brief Create an OrtEnv
-  * 
-  * \param[in] logging_level
-  * \param[in] logid
+  *
+  * \param[in] logging_level The log severity level.
```

Comment:
The comment makes me think the parameter should be severity_level vs logging_level. Not sure we can really fix it based on the type name.